core/src/main/java/org/pentaho/cassandra/driver/datastax/DriverCQLRowHandler.java

```diff
@@ -157,8 +157,9 @@ private Object readValue( ValueMetaInterface meta, Row row, int i ) {
         return row.getDouble( i );
       case ValueMetaInterface.TYPE_BIGNUMBER:
         return row.getDecimal( i );
+      case ValueMetaInterface.TYPE_TIMESTAMP:
       case ValueMetaInterface.TYPE_DATE:
-        return row.getDate( i );
+        return row.getTimestamp( i );
```

Comment:
Is this right? We want to call getTimestamp for both TIMESTAMP and DATE meta types?